src/CalendarAvailability.vue

```diff
@@ -33,23 +37,34 @@
 					{{ l10nEmptyDay }}
 				</span>
 			</div>
-			<button :key="`add-slot-${day.id}`"
+			<Button :key="`add-slot-${day.id}`"
 				:disabled="loading"
-				class="icon-add add-another button"
+				class="add-another button"
 				:title="l10nAddSlot"
-				@click="addSlot(day)" />
+				@click="addSlot(day)">
+				<template #icon>
+					<IconAdd :size="20" />
+				</template>
+			</Button>
 		</template>
 	</div>
 </template>
 
 <script>
-import NcDatetimePicker from '@nextcloud/vue/dist/Components/NcDatetimePicker.js'
+
+import DatetimePicker from '@nextcloud/vue/dist/Components/NcDatetimePicker'
+import Button from '@nextcloud/vue/dist/Components/NcButton'
+import IconAdd from 'vue-material-design-icons/Plus'
```

Comment:
`vue-material-design-icons` is not a dependency of this package at the moment

Reply:
ah sorry, you have already stated  inline material icon SVGs on the ticket. Fixing it asap

Reply:
add this dependency